tests/test_initialization.py

```diff
@@ -0,0 +1,7 @@
+import falguard
+
+
+def test_initialization():
+    falguard.Validator("tests/spec.yaml")
+    falguard.Validator("tests/spec.json")
+    falguard.Validator("tests/spec.yml")
```

Comment:
We need to run all the tests for every kind of spec file. I don't know how to make this combination yet. By all I mean everything from `test_error.py` and `test_ok.py`. Probably `falguard.Validator('tests/spec.json')` from `conftest.py` will have to go into `pytest.mark.parametrize`. Can you do it?

Reply:
Yep will dig on this!